config.py

```diff
@@ -39,6 +41,7 @@ class DevelopmentConfig(Config):
         "postgresql://timeless_user:timeless_pwd@localhost/timelessdb_dev")
     RESULT_BACKEND = "redis://localhost:6379"
     BROKER_URL = "redis://localhost:6379"
+    REDIS_HOST = "localhost"
```

Comment:
@ammaratef45 all these vars are looking at `redis`:
```python
    RESULT_BACKEND = "redis://localhost:6379"
    BROKER_URL = "redis://localhost:6379"
    REDIS_HOST = "localhost"
```
so I suggest to do the following:
```python
    REDIS_HOST =  "redis://localhost:6379"
    RESULT_BACKEND = REDIS_HOST
    BROKER_URL = REDIS_HOST
```

wdyt?

Reply:
@vryazanov I think you are right